### PR TITLE
fix: google font import and allignment of button

### PR DESCRIPTION
**Pull request by catosaurusrex2003**
Before google fonts were not working as expected because they were not imported at the top most line of our css. And this was causing CSS Parsing Order issue.

also fixed allignment of text inside button

**Changes in FIREFOX:**
image
👆 Before

image
👆 After

**Changes in CHROME:**
image
👆 Before

image
👆 After

I might not have been able to explain it correctly in the following issue #389 
But i hope the code in the PR makes sense.

Fixes issue : #389

### CODE REVIEW

**Comment by AceTheCreator** [comment]
Also the list of venues in the subnav is misaligned

**Comment by catosaurusrex2003** [comment]
ok.
i will check for all misalignment issues and fix it in this PR itself.

**Comment by catosaurusrex2003** [comment]
I have also noted this issue
image
look at the orientation of the + sign of the first item in the image above 👆. I think it should be X but it is slightly awkward.
image
shouldnt it be like this 👆 ? or is this intended behaviour.

**Comment by catosaurusrex2003** [comment]
@AceTheCreator  
/ptal

**Comment by ashmit-coder** [comment]
> I have also noted this minute thing image look at the orientation of the + sign of the first item in the image above 👆. I think it should be X but it is slightly awkward. image shouldnt it be like this 👆 ? or is this intended behaviour.

It does look better that way @catosaurusrex2003 go ahead with that change!

**Review by ashmit-coder — CHANGES_REQUESTED** [review]
Hey @catosaurusrex2003 , you seem to have removed the hover effect from the navbar.

Expected:
Screenshot from 2024-09-27 17-22-58

Current:
Screenshot from 2024-09-27 17-22-04

**Comment by catosaurusrex2003** [comment]
Nice observation @ashmit-coder 
Fixed it 👍

**Comment by ashmit-coder** [comment]
Hey @AceTheCreator are we moving ahead with this pr?

if yes then @catosaurusrex2003 could you please update the branch

**Comment by AceTheCreator** [comment]
waiting for your approval @ashmit-coder :)

**Comment by ashmit-coder** [comment]
> Hey @catosaurusrex2003 , you seem to have removed the hover effect from the navbar.
> 
> Expected: Screenshot from 2024-09-27 17-22-58
> 
> Current: Screenshot from 2024-09-27 17-22-04

@catosaurusrex2003 could you make the `Helsinki Finlad`. Appear on the same line as it was earlier

**Comment by catosaurusrex2003** [comment]
@ashmit-coder @AceTheCreator done

**Comment by ashmit-coder** [comment]
Hey @catosaurusrex2003 

image


fix this alignment issue in the Paris page

**Comment by ashmit-coder** [comment]
image


this as well.

**Comment by ashmit-coder** [comment]
image

This as well.

**Comment by ashmit-coder** [comment]
image

The ticket card also has some issues, do look into it.

**Comment by catosaurusrex2003** [comment]
I went through the whole website and now i dont think there are any discrepancies.

If any, pls let me know i will resolve them.

@ashmit-coder

**Comment by ashmit-coder** [comment]
Hey @catosaurusrex2003 could you check so for mobile view as well?

**Comment by catosaurusrex2003** [comment]
done @ashmit-coder 
i hope there isnt anything i missed out again 😆

**Review by AceTheCreator — APPROVED** [review]
LGTM! 🚀

**Comment by AceTheCreator** [comment]
CC @ashmit-coder

File: components/Buttons/button.js
Comment (catosaurusrex2003) [reply components/Buttons/button.js:9]: i am not able to locate any alignment issue.
whats the browser u using? and if possible can you attach the screenshot.

**Comment by ashmit-coder** [comment]
> CC @ashmit-coder

LGTM!

**Pull request merged**
